cli/src/commands/test.ts

```diff
@@ -0,0 +1,69 @@
+import { Command, flags } from "@oclif/command";
+import { safeParse } from "../common/safe-parse";
+import { runTest, TestFilter } from "../test-utils/test-runner";
+
+const ARG_API = "spot_contract";
+
+/**
+ * oclif command to test a spot contract
+ */
+export default class Test extends Command {
+  static description = "Test a Spot contract";
+
+  static examples = ["$ spot test api.ts -u http://localhost:3000"];
+
+  static args = [
+    {
+      name: ARG_API,
+      required: true,
+      description: "path to Spot contract",
+      hidden: false
```

Comment:
What is this for? Is the default behaviour `hidden: true`?

Reply:
`hidden: true` is the default and will hides the argument from the `help` menu